paper/paper.bib

```diff
@@ -0,0 +1,189 @@
+@UNPUBLISHED{Cellinese2021-gp,
```

Comment:
I don't think `@UNPUBLISHED` is the appropriate type here? And can you please add the DOI and/or URL that uses the DOI.

Reply:
Fixed to type `article` in 22857eef489c. Also, I didn't notice that all the DOIs/URLs had vanished! I've replaced the DOIs in 699193cc238e70, and they now [appear in the rendered PDF](https://whedon.theoj.org/preview?id=d85a050de1a01dfc9a8176d6).